tools/apiview/emitters/typespec-apiview/package.json

```diff
@@ -55,17 +55,16 @@
     "!dist/test/**"
   ],
   "peerDependencies": {
-    "@typespec/compiler": ">=0.67 <1.0",
-    "@typespec/versioning": ">=0.67 <1.0"
```

Comment:
Looks like this removal causing build to fail.

Reply:
Figured we didn't need versioning since I removed the versioning capabilities, but the tests need it... so it remains.